model/riscv_vext_regs.sail

```diff
@@ -79,7 +79,8 @@ mapping vreg_name = {
 function dirty_v_context() -> unit = {
   assert(sys_enable_vext());
   mstatus[VS] = extStatus_to_bits(Dirty);
-  mstatus[SD] = 0b1
+  if xlen == 32 then mstatus[SD_32] = 0b1
```

Comment:
Maybe add a `set_mstatus_sd()` function since there's already a `get` version and you use this twice.